README.md

```diff
@@ -318,8 +317,8 @@ Hello, Katy
 
 #### Short flags
 
-Options can often be set with a shorter version of their name, using short flags.
-Most popular example is `--force` option. Most CLIs also accept `-f` as a shorter version of the same option.
+Options can often beset with a shorter version of their name, using short flags.
```

Comment:
"`beset`" should be changed back to `be set`.

Reply:
Fixed